### DISCUSSION TITLE
How can I create a user with limited access (just can create a page and see his/her own pages)?

### DISCUSSION THREAD

**Discussion by MT-Dev06** [mixcore/mix.core#356] [Q&A] [answered]
Hi,
How can I create a user with limited access (just can create a page and see his/her own pages)
Thanks

**Comment by Smilefounder** [comment]
Hi @MT-Dev06 As I remember we have implemented this in our Portal SPA yet. Let me add this to our next update.

Related: #358 

cc @nhathoang989

**Comment by nhathoang989** [comment, marked as answer]
Hi @MT-Dev06 ,

You can add condition in ApiPageController & ApiPostController to filter Data by User like below:

![image](https://user-images.githubusercontent.com/5327699/108583412-db210000-736b-11eb-8c82-d75e469d12d3.png)
